test/contrib/automotive/gm/gmlanutils.uts

```diff
@@ -158,15 +165,19 @@ with new_can_socket0() as isocan, ISOTPSocket(isocan, sid=0x242, did=0x642, base
 
 
 = Positive, pending message from different service interferes while pending
+drain_bus(iface0)
 started = threading.Event()
 def ecusim():
     with new_can_socket0() as isocan, ISOTPSocket(isocan, sid=0x642, did=0x242, basecls=GMLAN) as isotpsock2:
         isotpsock2.sniff(count=1, timeout=1, started_callback=started.set)
         pending = GMLAN()/GMLAN_NR(requestServiceId=0x34, returnCode=0x78)
         isotpsock2.send(pending)
         wrongservice = GMLAN()/GMLAN_NR(requestServiceId=0x36, returnCode=0x78)
+        time.sleep(0.1)
```

Comment:
What's this for? If you need to add delay for a test to pass, you likely need to refactor it...